examples/multiple-projects-on-kubernetes-example/pom.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>io.dekorate</groupId>
+  <artifactId>multiple-project-parent-on-kubernetes-example</artifactId>
```

Comment:
I feel that `multimodule-project` would be more descriptive here.

Reply:
Renamed all the multiple words to `multimodule`